### PR TITLE
feat: Basic map component

### PR DESCRIPTION
**Pull request by alainajaffer**
A map should appear on the stops page

Testing:
Create .env.local file and then add the api key to it

### CODE REVIEW

File: site/components/form/MapBox.tsx
Comment (lj-kpmg) [inline site/components/form/MapBox.tsx:1]: I'd maybe just call this file Map.tsx

File: site/components/form/MapBox.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+interface MapBoxProps {
+    initialViewState: Partial<ViewState>;
+    style: CSSProperties;
+    mapStyle: string;
+}
+
+const MapBox = ({ initialViewState, style, mapStyle }: MapBoxProps): ReactElement => {
+    const mapboxAccessToken = process.env.MAP_BOX_ACCESS_TOKEN || "";
+
+    return (
```
Comment (lj-kpmg) [inline site/components/form/MapBox.tsx:13]: We probably only want to try and render the map if the mapboxAccessToken is non empty

File: site/pages/create-consequence-stops.page.tsx [hunk 2]
```diff
@@ -205,6 +205,16 @@ const CreateConsequenceStops = ({
                             inputId="stops"
                         />
 
+                        <MapBox
+                            initialViewState={{
+                                longitude: -100,
+                                latitude: 40,
+                                zoom: 3.5,
+                            }}
+                            style={{ width: 600, height: 400, marginBottom: 20 }}
```
[the remark this answers is not in the document]
Comment (alainajaffer) [reply site/pages/create-consequence-stops.page.tsx]: done

File: site/next.config.js [hunk 3]
```diff
@@ -28,6 +28,7 @@ const nextConfig = {
     poweredByHeader: false,
     env: {
         API_BASE_URL: process.env.API_BASE_URL,
+        MAP_BOX_ACCESS_TOKEN: process.env.MAP_BOX_ACCESS_TOKEN,
```
[the remark this answers is not in the document]
Comment (alainajaffer) [reply site/next.config.js:31]: done

File: site/components/form/MapBox.tsx (hunk 1 above)
Comment (alainajaffer) [reply site/components/form/MapBox.tsx]: done

File: site/components/form/MapBox.tsx
Comment (alainajaffer) [reply site/components/form/MapBox.tsx:1]: done

File: site/components/form/Map.tsx [hunk 4]
```diff
@@ -0,0 +1,23 @@
+import { CSSProperties, ReactElement } from "react";
+import Map, { ViewState } from "react-map-gl";
```
Comment (lj-kpmg) [inline site/components/form/Map.tsx:2]: We can update the name of this default import to MapBox to get rid of the lint warning and then change the name of the component from MapBox to Map

File: site/components/form/Map.tsx [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
+interface MapBoxProps {
```
Comment (lj-kpmg) [inline site/components/form/Map.tsx:4]: And this can be MapProps

Comment (alainajaffer) [reply site/components/form/Map.tsx]: done

File: site/components/form/Map.tsx (hunk 4 above)
Comment (alainajaffer) [reply site/components/form/Map.tsx]: done

**Review by lj-kpmg — APPROVED** [review]

**Pull request merged**